sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -320,7 +320,7 @@ def __len__(self):
     def _from_list(self, messages):
         for each in messages:
             if not isinstance(each, Message):
-                raise ValueError("Populating a message batch only supports iterables containing Message Objects.  "
+                raise ValueError("Only Message or iterable object containing Messages are accepted."
```

Comment:
```suggestion
                raise ValueError("Only Message or an iterable object containing Message objects are accepted."
```

(just to avoid potential confusion that there is a `Messages` type?)